source/telega/telegram/basic.d

```diff
@@ -319,7 +319,7 @@ struct Contact
     string phone_number;
     string first_name;
     Nullable!string last_name;
-    Nullable!string user_id;
+    Nullable!long user_id;
```

Comment:
```suggestion
    Nullable!uint user_id;
```
AFAIK user ID is not bigger than 32 bit unsigned, so this library uses uint everywhere